homeassistant/components/speedtestdotnet/__init__.py

```diff
@@ -5,34 +5,33 @@
 import speedtest
 import voluptuous as vol
 
-from homeassistant.components.sensor import DOMAIN as SENSOR_DOMAIN
-from homeassistant.const import CONF_MONITORED_CONDITIONS, CONF_SCAN_INTERVAL
+from homeassistant.config_entries import SOURCE_IMPORT
+from homeassistant.const import CONF_SCAN_INTERVAL
 import homeassistant.helpers.config_validation as cv
-from homeassistant.helpers.discovery import async_load_platform
-from homeassistant.helpers.dispatcher import dispatcher_send
+from homeassistant.helpers.dispatcher import async_dispatcher_send
 from homeassistant.helpers.event import async_track_time_interval
 
-from .const import DATA_UPDATED, DOMAIN, SENSOR_TYPES
+from .const import (
+    CONF_MANUAL,
+    CONF_SERVER_ID,
+    DATA_UPDATED,
+    DEFAULT_SCAN_INTERVAL,
+    DOMAIN,
+    SPEED_TEST_SERVICE,
+)
 
 _LOGGER = logging.getLogger(__name__)
 
-CONF_SERVER_ID = "server_id"
-CONF_MANUAL = "manual"
-
-DEFAULT_INTERVAL = timedelta(hours=1)
 
 CONFIG_SCHEMA = vol.Schema(
     {
         DOMAIN: vol.Schema(
             {
                 vol.Optional(CONF_SERVER_ID): cv.positive_int,
-                vol.Optional(CONF_SCAN_INTERVAL, default=DEFAULT_INTERVAL): vol.All(
-                    cv.time_period, cv.positive_timedelta
-                ),
-                vol.Optional(CONF_MANUAL, default=False): cv.boolean,
                 vol.Optional(
-                    CONF_MONITORED_CONDITIONS, default=list(SENSOR_TYPES)
-                ): vol.All(cv.ensure_list, [vol.In(list(SENSOR_TYPES))]),
+                    CONF_SCAN_INTERVAL, default=DEFAULT_SCAN_INTERVAL
+                ): vol.All(cv.time_period, cv.positive_timedelta),
+                vol.Optional(CONF_MANUAL, default=False): cv.boolean,
             }
```

Comment:
You should not remove `CONF_MONITORED_CONDITIONS`, it will be used in import configs